bliss/main.py

```diff
@@ -12,3 +12,7 @@ def main(cfg):
     else:
         raise KeyError
     task(cfg)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
hmm do you know why `codecov` now says that this line is not covered (I thought it would be used by both `test_generate.py` and `test_train.py`)

Reply:
Both tests import `main()` from`train.py` and `generate.py`. This file is not tested. I can remove these lines as this is not needed for our code to run. I add this just to debug everything conveniently.